docs/en/compute/Spark.md

```diff
@@ -90,19 +90,19 @@ of the input file `Input`.
 
 ### Access Data in Under Storage
 
-Alluxio supports transparently fetching the data from the under storage system,
+Alluxio transparently fetches data from the under storage system,
 given the exact path.
 For this section, HDFS is used as an example of a distributed under storage
 system.
 
 Put a file `Input_HDFS` into HDFS:
 
 ```console
-$ hdfs dfs -put -f ${ALLUXIO_HOME}/LICENSE hdfs://localhost:9000/alluxio/Input_HDFS
+$ hdfs dfs -copyFromLocal -f ${ALLUXIO_HOME}/LICENSE hdfs://localhost:9000/alluxio/Input_HDFS
```

Comment:
`-put` did not work for me. Data never actually was loaded into my hdfs cluster when I was testing. `-copyFromLocal` worked though.